ansible/roles/common/tasks/main.yml

```diff
@@ -13,14 +13,20 @@
 # limitations under the License.
 
 ---
+- name: set global vars
```

Comment:
Personally I don't think this step should be executed in ansible script, it should be executed manually.